modules/electromagnetics/include/auxkernels/SourceCurrentHeating.h

```diff
@@ -0,0 +1,40 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "AuxKernel.h"
+
+/**
+ *  Computes the heating due to the electic field in the form of
+ *  0.5 Re( J * E^* )
```

Comment:
Define these variables in this comment.

Reply:
Defined in [70a7473](https://github.com/idaholab/moose/pull/28759/commits/70a7473b7affe5e6fce2a562654a3f55bc306c9f)